.github/workflows/container_release.yaml

```diff
@@ -76,10 +72,15 @@ jobs:
 
       - name: List Google Cloud images
         run: |
-          poetry run rhelocator-updater gcp-images >> ./results/gcp_images.json
+          poetry run rhelocator-updater gcp-images >> ./data/temp-gcp-data.json
         env:
           GOOGLE_APPLICATION_CREDENTIALS: '${{ steps.google_auth.outputs.credentials_file_path }}'
 
+      - name: Merge JSON files
+        run: |
+          jq -s 'reduce .[] as $item ({}; . * $item)' ./data/* >> image-data.json
```

Comment:
oh cool! Once we get detailed azure image information, we can merge them with `jq` too! :smiley:

Reply:
Btw. good thing you commented on this :D ! I've noticed I forgot to add the path to the  pipe output :D